### DISCUSSION TITLE
Nested array fields using useWatch to watch the child in the nested array

### DISCUSSION THREAD

**Discussion by oliseh6123** [react-hook-form/react-hook-form#2983] [Q&A]
Hi @bluebill1049, i am trying to watch the child field in a nested array field from the level of the parent. I have a set of fields in nested array that i want to calculate the total sum on the parent level.
![nested array form](https://user-images.githubusercontent.com/36149597/93975249-12649400-fd6f-11ea-8565-0c3c3a586f16.png)
![nested array form2](https://user-images.githubusercontent.com/36149597/93975254-1395c100-fd6f-11ea-84f3-dbac4fffb6fa.png)

https://codesandbox.io/s/quiet-forest-pmykg?file=/src/nestedFieldArray.jsx:513-565 
here is the sandbox, i will appreciate your help

**Comment by bluebill1049** [comment]
hey @oliseh6123, try to pass `field` down to your `useWatch` call

`useWatch('watchName', field)` so you have a defaultValue for your `watch`, `watch` is not the same as `state`.

<img width="1000" alt="Screen Shot 2020-09-23 at 4 50 29 pm" src="https://user-images.githubusercontent.com/10513364/93976806-e53ed280-fdbc-11ea-99e2-9a4a2f076edb.png">

**Reply by oliseh6123** [reply]
Hi @bluebill1049 i get that and i will try it but as you can see from my picture i actually got the desire result for the first form but the answer which is 30 is still reflecting in the second form
